### PR TITLE
initial implementation of subscriptions

### PR DESCRIPTION
**Pull request by helfer**

### CODE REVIEW

File: api/index.js [hunk 1]
```diff
@@ -123,3 +123,36 @@ passport.use(new GitHubStrategy(gitHubStrategyOptions, (accessToken, refreshToke
...
+  response.writeHead(404);
+  response.end();
+});
+
+httpServer.listen(WS_PORT, () => console.log( // eslint-disable-line no-console
+  `Websocket Server is now running on http://localhost:${WS_PORT}`
+));
+
+// eslint-disable-next-line
+new Server(
```
Comment (stubailo) [inline api/index.js]: I'd rename this to `WebsocketServer` in the import or something

File: api/index.js [hunk 2]
```diff
@@ -123,3 +123,36 @@ passport.use(new GitHubStrategy(gitHubStrategyOptions, (accessToken, refreshToke
...
+  {
+    subscriptionManager,
+    onSubscribe: (msg, params) => {
+      const gitHubConnector = new GitHubConnector({
```
Comment (stubailo) [inline api/index.js]: I'd add a comment that this is setting the context

File: api/index.js [hunk 3]
```diff
@@ -1,14 +1,31 @@
 import path from 'path';
 import express from 'express';
 import { apolloExpress, graphiqlExpress } from 'apollo-server';
+<<<<<<< 2525b3968d283166c65f7685d233ec82206ecc91
```
Comment (stubailo) [inline api/index.js]: Merge conflict

**Comment by rricard** [comment]
Great code to read in order to get started, thanks @helfer 👍 !

**Pull request merged**